packages/regl-worldview/src/commands/FilledPolygons.js

```diff
@@ -32,7 +32,7 @@ function flatten3D(points: Vec3[]): Float32Array {
   return array;
 }
 
-function getEarcutPoints(points: Vec3[]): Vec3[] {
+function getEarcutPoints(points: Vec3[]): (Vec3 | Point)[] {
```

Comment:
Might help to use `$ReadOnlyArray<Vec3|Point>` elsewhere to avoid having to change this type.